robot/vision.py

```diff
@@ -209,13 +210,29 @@ class RoboConUSBCamera(Camera):
     def __init__(self,
                  start_res=(1296, 736),
                  focal_lengths=None):
-        self._cv_capture = cv2.VideoCapture(0)
+        self._source = self.find_usb_cam()
+        if self._source == None:
+            raise Exception("No USB camera detected")
```

Comment:
MAYBE: it might be useful to let the user know what their next steps are to fix for this. something like `make sure it is plugged in`.

Reply:
Not a bad idea, here's a patch
```suggestion
            raise Exception("No USB camera detected, please make sure it's plugged in")
```

Reply:
Thanks